app/src/main/java/org/simple/clinic/settings/SettingsModule.kt

```diff
@@ -21,7 +21,12 @@ class SettingsModule {
   }
 
   @Provides
-  fun provideSettingsRepository(): SettingsRepository {
-    return SettingsRepositoryImpl()
+  fun provideSettingsRepository(
+      @Named("preference_user_selected_locale") userSelectedLocalePreference: Preference<Optional<Locale>>
+  ): SettingsRepository {
+    return SettingsRepositoryImpl(
+        userSelectedLocalePreference = userSelectedLocalePreference,
+        supportedLanguages = emptyList()
```

Comment:
Curious, why is supported languages an empty list?

Reply:
It's empty for now because it's not used in production. The final commit in this PR adds the actual languages we currently support.